docs/cql/functions.rst

```diff
@@ -228,6 +230,65 @@ A number of functions are provided to “convert” the native types into binary
 takes a 64-bit ``blob`` argument and converts it to a ``bigint`` value. For example, ``bigintAsBlob(3)`` is
 ``0x0000000000000003`` and ``blobAsBigint(0x0000000000000003)`` is ``3``.
 
+.. _udfs:
+
+User-defined functions
+~~~~~~~~~~~~~~~~~~~~~~
+
+User-defined functions (UDFs) execute user-provided code in ScyllaDB. Supported languages are currently Lua and WebAssembly.
+
+UDFs are part of the Cassandra schema, and are automatically propagated to all nodes in the cluster.
```

Comment:
Maybe "Scylla" instead of "Cassandra"? Or "keyspace" (since, if I remember correctly, UDFs are per keyspace?)